develop/dev-guide-choose-driver-or-orm.md

```diff
@@ -29,9 +29,10 @@ Support level: **Full**
 
 You can follow the [MySQL documentation](https://dev.mysql.com/doc/connector-j/en/) to download and configure a Java JDBC driver. It is recommended to use MySQL Connector/J 8.0.33 or later with TiDB v6.3.0 and newer.
 
-> **Tip:**
+> **Note:**
 >
-> There is a [bug](https://bugs.mysql.com/bug.php?id=106252) in the Connector/J 8.0 versions before 8.0.32, which might cause threads to hang when using TiDB versions earlier than v6.3.0. To avoid this issue, it is recommended that you use either MySQL Connector/J 8.0.32 or a later version, or the TiDB JDBC (see the *TiDB-JDBC* tab).
+> - There is a [bug](https://bugs.mysql.com/bug.php?id=106252) in the Connector/J 8.0 versions before 8.0.32, which might cause threads to hang when using TiDB versions earlier than v6.3.0. To avoid this issue, it is recommended that you use either MySQL Connector/J 8.0.32 or a later version, or the TiDB JDBC (see the *TiDB-JDBC* tab).
+> - When using MySQL Connector/J 8.0 with TiDB v7.5.0, it is recommended to set the TiDB configuration item [`server-version`](https://docs.pingcap.com/tidb/v7.5/tidb-configuration-file#server-version) to `"5.7.25-TiDB-v7.5.0"`. MySQL Connector/J attempts to access the `information_schema.KEYWORDS` table if the TiDB server reports a version of MySQL 8.0.11 or later. However, this table is not present in TiDB v7.5.0.
 
```

Comment:
@dveeden, please help confirm whether https://github.com/pingcap/tidb/pull/48807 will be cherry-picked to v7.5.  If yes, the above change looks okay because the issue will be fixed for v7.5.1.

Reply:
It is currently not clear if this will be backported or not. I'm discussing this with various people.

Reply:
Got it. If it is not backported in v7.5.1, we'll also need to include v7.5.1 in this note when v7.5.1 is published.